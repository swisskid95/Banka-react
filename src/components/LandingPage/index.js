import React from 'react';
import { Link } from 'react-router-dom';

const LandingPage = () => {
  return (
    <main className="container__entry">
      <div className="landing__page">
        <div className="section-wrapper">
          <section className="main-section">
            <h1 className="logo--big">Banka</h1>
            <h2 className="first-text">Banking made easier</h2>
            <p className="second-text">
              Banka helps you track all transaction on your account(s)
            </p>
            <Link to="/sign-up" className="cta">
              SIGN-UP{' '}
            </Link>
          </section>
          <section className="infograph-section">
            <img
              className="monitor"
              src="../../src/assets/images/infograph.svg"
              alt="infograph"
            />
          </section>
        </div>
      </div>
    </main>
  );
};

export default LandingPage;
