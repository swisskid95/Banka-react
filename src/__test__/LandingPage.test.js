/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import LandingPage from '../components/LandingPage';

it('should render correctly', () => {
  const wrapper = shallow(<LandingPage />);

  expect(wrapper).toMatchSnapshot();
});
