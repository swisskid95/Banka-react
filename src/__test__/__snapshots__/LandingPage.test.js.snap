// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<main
  className="container__entry"
>
  <div
    className="landing__page"
  >
    <div
      className="section-wrapper"
    >
      <section
        className="main-section"
      >
        <h1
          className="logo--big"
          data-testid="logo"
        >
          Banka
        </h1>
        <h2
          className="first-text"
        >
          Banking made easier
        </h2>
        <p
          className="second-text"
        >
          Banka helps you track all transaction on your account(s)
        </p>
        <Link
          className="cta"
          to="/sign-up"
        >
          SIGN-UP
           
        </Link>
      </section>
      <section
        className="infograph-section"
      >
        <img
          alt="infograph"
          className="monitor"
          src="/src/assets/images/infograph.svg"
        />
      </section>
    </div>
  </div>
</main>
`;
